keps/sig-apps/85-Graduate-PDB-to-Stable/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: KEP Template
+kep-number: 85
+authors:
+  - "@bsalamat"
+  - "@mortent"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-scheduling
+status: implementable
+creation-date: 2019-03-18
+reviewers:
+  - "@liggitt"
+  - "@kow3ns"
+  - "@soltysh"
+  - "@janetkuo"
+  - "@wojtek-t"
+approvers:
+  - "@kow3ns"
+  - "@liggitt"
+  - "@wojtek-t"
+prr-approvers:
+  - "@wojtek-t"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
```

Comment:
1.21

Reply:
Also - once you switch this, you will be forced to add an appropriate new file under keps/prod-readiness:

see https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-api-machinery/1040.yaml as an example

Reply:
Updated the version and added the PRR approval file.